spec/unit/logstash_input_spec.rb

```diff
@@ -26,7 +26,7 @@
   end
 
   describe "plugin register" do
-    let(:config) {{ "port" => 123 }}
+    let(:config) {{ }}
 
     let(:registered_plugin) { plugin.tap(&:register) }
 
```

Comment:
Missing coverage for the port and host, which could be something like:

```suggestion
    context "configuring port" do
      let(:config) { { "ssl_enabled" => false } } # minimum config
      context "default value" do
        it "reflects default port" do
          expect(registered_plugin.port).to eq 9800
        end
      end
      context "explicitly configured" do
        let(:config) { super().merge("port" => 1734) }
        it "reflects the configured port" do
          expect(registered_plugin.port).to eq 1734
        end
      end
    end

    context "configuring host" do
      let(:config) { { "ssl_enabled" => false } } # minimum config
      context "default value" do
        it "reflects default host" do
          expect(registered_plugin.host).to eq "0.0.0.0"
        end
      end
      context "explicitly configured" do
        let(:config) { super().merge("host" => "10.0.3.218") }
        it "reflects the configured host" do
          expect(registered_plugin.port).to eq "10.0.3.218"
        end
      end
    end

```